modules/osdk-managing-psa-for-operators-with-escalated-permissions.adoc

```diff
@@ -6,7 +6,7 @@
 [id="osdk-managing-psa-for-operators-with-escalated-permissions.adoc_{context}"]
 = Managing pod security admission for Operator workloads that require escalated permissions
 
-If your Operator project requires escalated permissions to run, you cannot rely on pod security admission synchronization to label your Operator's namespaces. You must set your Operator's namespace label with the appropriate pod security level.
+If your Operator project requires escalated permissions to run, you cannot rely on pod security admission synchronization to label your Operator's namespaces. You must set your Operator's namespace label with the appropriate pod security level in your Operators custer service version (CSV).
```

Comment:
hm.  This isn't right.

1) you can still rely on PSA label syncing.  Your operator pod is presumably running with a service account that has appropriate SCC permissions(escalated or otherwise). That in turn means the NS should get appropriate PSA labels synced.  The docs below cover the part about granting the SCC permissions, so the documented workflow is ok.

2) if you do need to label the NS manually, you can't specify that in the CSV.  Really the only solution at that point is the admin needs to do it manually (potentially as part of creating the NS for the operator installation).  But labelsyncing should cover most situations.

Reply:
Addressed in 6946ad6. I made the abstract a bit more generic and removed the admonition on namespace labelling since it didn't seem to fit here. Does that seem right to you?